kore/test/Test/Kore/BugReport.hs

```diff
@@ -82,6 +90,40 @@ test_parse =
             actual = expect >>= parseKoreLogOpts . unparseKoreLogOptions
         getParseResult expect === getParseResult actual
 
+test_bugReportOption :: TestTree
+test_bugReportOption =
+    testGroup
+        "Parse BugReportOption"
```

Comment:
Our test discovery tool will name the test group correctly if you do this instead:

```suggestion
test_Parse_BugReportOption :: [TestTree]
test_Parse_BugReportOption =
```